zip-0209.rst

```diff
@@ -0,0 +1,48 @@
+::
+
+  ZIP: 209
+  Title: Prohibit Negative Shielded Value Pool
+  Author: Sean Bowe <sean@z.cash>
+  Category: Consensus
+  Created: 2019-02-25
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The "Sprout value pool balance", as implied by section 4.11 of the Zcash Protocol Specification [#protocol]_, is the sum of all previous transactions' ``vpub_old`` fields subtracted by the sum of all previous transactions ``vpub_new`` fields.
+
+The "Sapling value pool balance", as implied by section 4.12 of Zcash Protocol Specification [#protocol]_, is the negation of the sum of all previous transactions' ``valueBalance`` fields.
+
+Abstract
+========
+
+This proposal defines how the consensus rules are altered such that blocks which produce negative shielded value pools are prohibited.
+
+Motivation
+==========
+
+It is possible for nodes to monitor the total amount of coins that are shielded or unshielded in each of the Sprout or Sapling shielded transaction protocols. If the amount of coins that are unshielded exceed the amount that are shielded, a balance violation has occurred in the corresponding shielded transaction protocol.
+
+It would be preferable for the network to reject blocks that result in the aforementioned balance violation. However, nodes do not currently react to such an event. Remediation may therefore require chain rollbacks and other disruption.
```

Comment:
This is a little bit clunky, here is a suggested rewording:

Rather than allow blocks which result in such a balance violation, it would be preferable for the network to reject them. As nodes do not currently react to this, remediation to such an event may require chain rollbacks and other disruptions.

Reply:
:man_shrugging: I am okay with the current wording. This rewording uses "them" and "this" when there are many objects (blocks, balance violations, the network, etc.)